charts/seed-monitoring/charts/core/charts/prometheus/rules/kube-apiserver.rules.yaml

```diff
@@ -49,6 +49,19 @@ groups:
     expr: histogram_quantile(0.5, rate(apiserver_request_duration_seconds_bucket[5m]))
     labels:
       quantile: "0.5"
+  # TODO remove with better metrics in the future (wyb1)
+  - record: shoot:apiserver_watch_duration:quantile
+    expr: histogram_quantile(0.2, sum(rate(apiserver_request_duration_seconds_bucket{verb="WATCH",resource=~"configmaps|deployments|secrets|daemonsets|services|nodes|pods|namespaces|endpoints|events|statefulsets|clusterroles|roles"}[5m])) by (le,scope,resource))
```

Comment:
In general the list of resources looks good. Still, two remarks:
- I think we can remove `events`, we have hardly any watches on events and it shouldn't be a problem if those watches don't last very long.
- I'm wondering if we should also collect metrics for our extension resources on the Seeds. If we have problems with short watches there, this will likely result in failures. As we are anyways in the evaluation phase for those metrics, can we add them and see what we will record over the next few weeks? My proposal would be to add a second recording rule with something like `group=~".+garden.+"`, if that's possible.

WDYT?

Reply:
I will remove `events`. Regarding the extensions, do you mean that we should have the same metrics again, but it is a subset containing only requests from the group `.+garden.+`?

Reply:
Yes, basically I mean the same metric name with the same source, only with a selection for gardener extension/gardener-resource-manager/dns resources.